tests/covalent_tests/results_manager_tests/results_test.py

```diff
@@ -58,24 +60,30 @@ def db():
     )
 
 
+le = LocalExecutor(log_stdout="/tmp/stdout.log")
+
```

Comment:
(non-blocking) unnecessary empty lines

Reply:
This one was actually added by pre-commit.